site/src/patterns/side-scrolling-container.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Side scrolling container
+description: Side scrolling container pattern in the Atlas Design System
+template: standard
+---
+
+# Side scrolling container
+
+WIP
+
+```html
+<div class="scroll-horizontal scroll-snap-container display-flex padding-bottom-xs">
+	<div class="padding-sm border scroll-snap-item" id="one">
+		<h3 class="font-size-lg">1</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="two">
+		<h3 class="font-size-lg">2</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="three">
+		<h3 class="font-size-lg">3</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="four">
+		<h3 class="font-size-lg">4</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="five">
+		<h3 class="font-size-lg">5</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="six">
+		<h3 class="font-size-lg">6</h3>
+		<p>Description text is here and goes on for a little while.</p>
+	</div>
+	<div class="padding-sm border scroll-snap-item" id="seven">
+		<h3 class="font-size-lg">7</h3>
+		<p>Description text is here and goes on for a little while. <br /> Even more text.</p>
+	</div>
+</div>
+<div class="jump-links">
```

Comment:
I think .button > .button should work file. `border-radius` should be removed.

Reply:
updated